util/chplenv/chpl_machine.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python
+import optparse
+import os
+import sys
+
+chplenv_dir = os.path.dirname(__file__)
+sys.path.insert(0, os.path.abspath(chplenv_dir))
```

Comment:
Sorry, https://github.com/chapel-lang/chapel/pull/11725 changed things out from under you. Please remove the path hacking (and os/sys includes if they're not needed elsewhere.) Ditto for chpl_bin_subdir.py 

printchplenv is also now a bash wrapper, so you'll have to update printchplenv.py instead